cloud-pack/README.md

```diff
@@ -4,7 +4,7 @@
 
 This directory provides an uber [Apache Jackrabbit FileVault](https://jackrabbit.apache.org/filevault/index.html) package with the Adobe Experience Manager (AEM) Connector for [StreamX](https://www.streamx.dev/StreamX) (*"Cloud Package with AEM Connector"*). The term _uber_ signifies that the package includes all necessary dependencies and OSGi configurations required for the AEM Connector to function within AEM.
 
-The Cloud Package with AEM Connector is compatible exclusively with AEM as a Cloud Service.
+The Cloud Package with AEM Connector is compatible with both AEM as a Cloud Service and AEM 6.5.22+.
```

Comment:
just add the note we discussed:
- why not supported in lover version
- what to do for lower versions (check cloud-pack setup and do not use envs ingestion)